package.json

```
@@ -27,7 +27,7 @@
   "license": "SEE LICENSE IN https://tc39.es/ecma262/#sec-copyright-and-software-license",
   "homepage": "https://tc39.es/ecma262/",
   "dependencies": {
-    "ecmarkup": "9.6.1"
+    "ecmarkup": "9.6.3"
```

```suggestion
    "ecmarkup": "^9.6.3"
```
will merge as-is, but we should fix this next time it's bumped